.drone.yml

```diff
@@ -0,0 +1,85 @@
+workspace:
+  base: /go
+  path: src/github.com/src-d/landing
+
+branches: master
+
+pipeline:
+
+  build:
+    image: golang:1.8
+    commands:
+      - apt-get update
+      - apt-get install -y make bash curl git
+      - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.4/install.sh | sh
+      - . ~/.nvm/nvm.sh
+      - nvm install 8
+      - npm install -g yarn
+      - npm rebuild node-sass
+      - go get -v -t ./...
+      - make build
+      - make packages
+    debug: true
```

Comment:
The [travis old file did `make package-hugo-generated`](https://github.com/src-d/landing/pull/159/files#diff-354f30a63fb0907d4ad57269548329e3L40); I think that @erizocosmico did it to prepare the artifacts being published with the release, that seemed to be used during the deploy stage, is it no longer needed?

Reply:
As I understood @mcuadros, we won't publish landing releases on github anymore.